keyboards/preonic/keymaps/rs/config.h

```diff
@@ -0,0 +1,40 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```

Comment:
The build script includes this file via the keyboard's config.h file. This line should be deleted.

```suggestion
```